pydatastructs/graphs/algorithms.py

```diff
@@ -697,7 +697,7 @@ def shortest_paths(graph: Graph, algorithm: str,
         The algorithm to be used. Currently, the following algorithms
         are implemented,
 
-        'bellman_ford' -> Bellman-Ford algorithm as given in [1].
+        'bellman_ford' -> Bellman-Ford algorithm as given in [1], with a queue to improve performance on sparse graphs.
```

Comment:
```suggestion
        'bellman_ford' -> Bellman-Ford algorithm as given in [1]
```

Implementation detail aren't required to be mentioned.